docs/js-core-1/week-4/lesson.md

```diff
@@ -163,10 +163,10 @@ Did you notice how calling the concat method did not change `arr`? This is becau
 If you want to use the array returned by calling `.concat()` you should store it in a new variable.
 
 ```js
-const array = [1, 2, 3];
-const newArray = array.concat(4);
+const temp = [1, 2, 3];
```

Comment:
> A good variable name quickly explains what it represents to anyone reading the code. Try to describe what the variable is or what it does within in code. ~[ Style Guide](https://syllabus.codeyourfuture.io/guides/code-style-guide#describe-what-the-variable-isdoes)

In this case, we are teaching a beginner how to concatenate two arrays, so it is easy to think that we should say `array` or temp. But really we have to make it clearer to the learner what the variables are **doing**, to make this a richer learning experience for them, as well as being consistent with the rules we set for their coursework.

Here's the context :

> Did you notice how calling the concat method did not change arr? This is because concat, like most array methods, returns a new array; it does not alter the array you called the method on.

> If you want to use the array returned by calling .concat() you should store it in a new variable.

```
const exampleArrayOfNumbers = [1, 2, 3];
const newReturnedArray = exampleArrayOfNumbers.concat(4);

console.log(newArrayToReturn); // logs [1, 2, 3, 4]
```

Reply:
okay got it I will change all the lines which are violating style guide
I just though that you only want to change on particular  lines so I changed that

Reply:
@SallyMcGrath check it